sdk/messaging/azservicebus/README.md

```diff
@@ -267,7 +265,7 @@ deadLetterReceiver, err := client.NewReceiverForQueue("<queue>",
 //   })
 ```
 
-To see some example code for receiving messages using the Receiver see the ["Receive messages"](#receive-messages) sample.
+To see some example code for receiving messages using the receiver, see the [Receive messages](#receive-messages) example.
```

Comment:
The capitalization here is because it's referencing the `Receiver` type. Should I make it more distinctive using backticks?

Reply:
Yes please, any type or a function call should be encapsulated with backticks for better readability and compliance with Microsoft copy style.

Reply:
Fixed capitalization and formatted with backgticks, thx